src/styles/main.scss

```
@@ -1,3 +1,235 @@
 body {
   margin: 0;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+}
+
+.calendar {
+  width: 790px;
+  height: 400px;
+  flex-wrap: wrap;
+  display: flex;
+  gap: 1px;
+  align-items: center;
+  justify-content: flex-start;
+  box-sizing: border-box;
+
+  @for $number from 1 through 30 {
+    &__day:nth-child(#{$number})::before {
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: #000;
+    }
+  }
+
+  @for $number from 1 through 30 {
+    &__day-start-day-mon:nth-child(#{$number})::before {
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: #000;
+    }
+  }
+
+  @for $number from 1 through 30 {
+    &__day-start-day-tue:nth-child(#{$number})::before {
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: #000;
+    }
+  }
+
+  @for $number from 1 through 30 {
+    &__day-start-day-wed:nth-child(#{$number})::before {
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: #000;
+    }
+  }
+
+  @for $number from 1 through 30 {
+    &__day-start-day-thu:nth-child(#{$number})::before {
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: #000;
+    }
+  }
+
+  @for $number from 1 through 30 {
+    &__day-start-day-fri:nth-child(#{$number})::before {
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: #000;
+    }
+  }
+
+  @for $number from 1 through 30 {
+    &__day-start-day-sat:nth-child(#{$number})::before {
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: #000;
+    }
+  }
+
+  @for $number from 1 through 30 {
+    &__day-start-day-sun:nth-child(#{$number})::before {
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: #000;
+    }
+  }
```

content doesn`t depends on day of week
```suggestion
  @for $index-day from 1 through 31 {
    &__day:nth-child(n + #{$index-day}):before {
      content: "#{$index-day}";
    }
  }
```